packages/babel-plugin-transform-object-rest-spread/src/index.js

```diff
@@ -39,7 +39,11 @@ export default function ({ types: t }) {
           args.unshift(t.objectExpression([]));
         }
 
-        path.replaceWith(t.callExpression(file.addHelper("extends"), args));
+        const helper = file.opts.polyfill === false ?
```

Comment:
Can we make this throw if it is passed the wrong type? E.g.

    if (polyfill === undefined) polyfill = true;
    if (typeof polyfill !== 'boolean') throw new Error